test-site/src/App.tsx

```diff
@@ -14,6 +14,7 @@ const config: HeadlessConfig = {
   analyticsConfig: {
     endpoint: "https://www.dev.us.yextevents.com/accounts/me/events",
   },
+  saveToLocalStorage: true,
```

Comment:
nit: can we remove this? often time in testing, we prefer not saving to local storage to avoid having to keep clearing the storage